packages/server/src/runHttpQuery.ts

```diff
@@ -81,29 +82,32 @@ function jsonParsedSearchParamIfSpecifiedOnce(
 }
 
 function fieldIfRecord(
-  o: Record<string, any>,
+  o: Record<string, unknown>,
   fieldName: string,
-): Record<string, any> | undefined {
-  if (isStringRecord(o[fieldName])) {
-    return o[fieldName];
+): Record<string, unknown> | undefined {
+  const value = o[fieldName];
+  if (isStringRecord(value)) {
+    return value;
   }
   return undefined;
 }
 
-function isStringRecord(o: any): o is Record<string, any> {
-  return o && typeof o === 'object' && !Buffer.isBuffer(o) && !Array.isArray(o);
+function isStringRecord(o: unknown): o is Record<string, unknown> {
+  return (
+    !!o && typeof o === 'object' && !Buffer.isBuffer(o) && !Array.isArray(o)
+  );
 }
 
-function isNonEmptyStringRecord(o: any): o is Record<string, any> {
+function isNonEmptyStringRecord(o: unknown): o is Record<string, unknown> {
   return isStringRecord(o) && Object.keys(o).length > 0;
 }
 
-function ensureQueryIsStringOrMissing(query: any) {
+function ensureQueryIsStringOrMissing(query: unknown) {
   if (!query || typeof query === 'string') {
     return;
   }
   // Check for a common error first.
-  if (query.kind === 'Document') {
+  if ((query as any).kind === 'Document') {
```

Comment:
Not your change but `'Document'` should be `Kind.Document`

Reply:
well, Kind.DOCUMENT. I guess, although we're recognizing a plain JSON object, not an actual enum... but sure.